### PR TITLE
Update Nuxt, add Vuetify, add NavBar example component

### PR DESCRIPTION
**Pull request by katatsu12**
Signed-off-by: den <katatsu12@gmail.com>

### CODE REVIEW

File: components/NavBar.vue [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      v-spacer
+      v-toolbar-items
+        v-btn(text)
+          nuxt-link(to='/') Home
+        v-btn(text)
+          nuxt-link(to='/about') About
+</template>
+
+<style scoped lang="sass">
+a
```
Comment (Alexey-Lukin) [inline components/NavBar.vue]: do we need indentation here?

File: components/NavBar.vue [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  text-decoration: none
+  color: black
```
Comment (Alexey-Lukin) [inline components/NavBar.vue]: are we placing styles in alphabet order?

File: layouts/default.vue [hunk 3]
```diff
@@ -1,55 +1,17 @@
...
-  display: inline-block;
-  border-radius: 4px;
-  border: 1px solid #35495e;
-  color: #35495e;
-  text-decoration: none;
-  padding: 10px 30px;
-  margin-left: 15px;
-}
+<script>
+import NavBar from '@/components/NavBar.vue'
```
Comment (Alexey-Lukin) [inline layouts/default.vue]: do we need indentation here?

File: package.json [hunk 4]
```diff
@@ -15,9 +15,10 @@
     "vuetify": "^2.0.0"
   },
   "devDependencies": {
+    "@nuxtjs/vuetify": "^1.9.0",
```
Comment (Alexey-Lukin) [inline package.json]: may be it should be in dependencies section?

File: package.json [hunk 5]
```diff
@@ -15,9 +15,10 @@
...
     "coffee-loader": "^0.9.0",
     "coffeescript": "^2.4.1",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.13.0",
```
Comment (Alexey-Lukin) [inline package.json]: why we have both yarn.lock and package.lock in codebase?

File: pages/about.vue [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<template lang="pug">
+  h1 About page
+</template>
+
```
Comment (Alexey-Lukin) [inline pages/about.vue]: remove extra line

File: layouts/default.vue (hunk 3 above)
Comment (katatsu12) [reply layouts/default.vue]: @Alexey-Lukin In official doc they use same indentation, but I like your variant more 
https://nuxtjs.org/examples

File: package.json (hunk 5 above)
Comment (katatsu12) [reply package.json]: yarn.lock removed

File: package.json (hunk 4 above)
Comment (katatsu12) [reply package.json]: @Alexey-Lukin It's default result installation Vuetify.js

**Comment by katatsu12** [comment]
@Alexey-Lukin Thank you for your time, your comments very important for us :+1:

**Pull request merged**